it/services/VatDetailsServiceISpec.scala

```diff
@@ -72,16 +75,15 @@ class VatDetailsServiceISpec extends IntegrationBaseSpec {
 
     }
 
-    "the user has no outstanding obligations" should {
+    "the user has no outstanding obligations or payments" should {
```

Comment:
Can we have scenarios for obligations and no payments and no obligations and payments please